Interface/ui_title_text.csv

```diff
@@ -54,41 +54,41 @@ UseAgreement_Message_ps4_break2#0,"""第11条（コミュニティ機能等）<b
-LoginIdSelect_LinkSegaId#0,"""SEGA IDと連携する"""
-LoginIdSelect_SegaId_Link#0,"""SEGA IDへ移行する"""
-Title#0,"""ログインメニュー"""
-TitleVita#0,"""タイトルメニュー"""
+LoginIdSelect_Sega#0,"""Iniciar Jogo"""
+LoginVita#0,"""Iniciar Jogo"""
+LoginIdSelect_UserInfo#0,"""Mudar informacao de Usuario"""
+LoginIdSelect_UserInfo_v#0,"""Mudar informacao de Usuario (Web)"""
+LoginIdSelect_AR#0,"""Foto de perfil do Personagem"""
+LoginIdSelect_CharaCreate#0,"""Criar Personagem"""
+LoginIdSelect_Credit#0,"""Creditos"""
+LoginIdSelect_GameEnd#0,"""Sair do Jogo"""
+LoginIdSelect_OnlineManual#0,"""Manual Online"""
+LoginIdSelect_ReturnTitle#0,"""Voltar a Tela Inicial"""
+LoginIdSelect_LinkSegaId#0,"""Link SEGA ID"""
+LoginIdSelect_SegaId_Link#0,"""Migrar para SEGA ID"""
+Title#0,"""Menu de Login"""
+TitleVita#0,"""Menu Principal"""
 UserInfoURL#0,"""https://USER-INFO-CHANGE-URL"""
 UserInfoURLVita#0,"""https://USER-INFO-CHANGE-URL"""
-FirstPlay_Message#0,"""『ファンタシースターオンライン2』へようこそ！<br><br>このゲームでは、「シップ」と呼ばれるサーバーの<br>どれかを選び、キャラクターを登録します。<br><br><yellow>■はじめてプレイする方は<c><br>「はじめてゲームを開始する」を選んでください。<br><br><yellow>■お友達招待システムで招待されている方は<c><br>お友達から指定されたシップを選択してください。<br><br><yellow>■すでにプレイしている方は<c><br>プレイ中のシップを選択してください。"""
-IssueUserIdName_Message#0,"""あなたのプレイヤーIDネーム：<br>\u3000\u3000<yellow>$(0)<c><br><br>プレイヤーIDネームとは、<br>プレイヤー個別に割り当てられる文字列で<br>コミュニケーション時などに使う名前です。<br><br>この名前は、ゲーム開始後に<br><yellow>好きな名前に変更<c>できます（初回無料）。<br><br>変更する場合は、シップ選択直後の<br>スタートメニュー＞サポートメニューから<br>「プレイヤーIDネーム変更」で可能です。"""
-Recommand_Message#0,"""はじめからゲームを開始します。<br><br>これから作成するキャラクターは、<br><br><yellow>「 $(0) 」<c><br><br>というシップ（サーバー）に登録されますので、<br>今後はこのシップを選択し、プレイしてください。<br><br>※お友達招待システムで招待されている方や、<br>\u3000別のシップを希望する方は、「戻る」を選択し<br>\u3000任意のシップを選択してください。"""
-Recommand_Title_Vita#0,"""『ファンタシースターオンライン2』へようこそ！"""
-Recommand_Message_Vita#0,"""このゲームでは、<green>「シップ」<c>と呼ばれるサーバーの<br>いずれかを選び、キャラクターを作成します。<br>作成したキャラクターは、選択したシップでのみ<br>プレイすることが可能です。<br><br>初めてプレイされる方は<green>「$(0)」<c>を<br>選択すると、より快適にプレイして頂けます。<br><br><yellow>別のシップを希望する方や、<br>お友達招待システムで招待されている方は、<br>任意のシップを選択してください。<c><br><br><red>※シップが別のキャラクターとは交流できません。<br>\u3000シップ移動にはアークスキャッシュ(AC)が必要です。<c>"""
-Recommand_Start#0,"""ゲームを開始する"""
-Recommand_Back#0,"""戻る"""
-LoginCloud#0,"""ゲームスタート"""
-LoginIdSelect_es#0,"""『PSO2es』との連動"""
-LoginIdSelect_es_Title#0,"""『ＰＳＯ２ｅｓ』とキャラクターを連動するには"""
-LoginIdSelect_es_Message#0,"""『ＰＳＯ２ｅｓ』のキャラクターで『ＰＳＯ２』をプレイするには<br>『ＰＳＯ２ｅｓ』から連動処理を行う必要があります。<br><br>『ＰＳＯ２ｅｓ』ロビー画面の「ＰＳＯ２連動」メニューから<br>「今すぐ連動する」ボタンをタッチし<br>メニューの説明にしたがって、連動を完了してから<br>『ＰＳＯ２』にログインしてください。<br><br>\u3000<&URL pso2.jp_es_players_guide_beginners>"""
+FirstPlay_Message#0,"""Bemvindo ao Phantasy Star Online 2!<br><br>Neste jogo voce pode selecionar uma<green>\""Nave""<c> e registre seu perosnagem.<br><br><yellaow>■Para Iniciantes:<c><br>Por favor selecione\""Iniciando o jogo pela primeira vez\"".<br><br><yellow>■Para aqueles que foram convidados por amigos:<c><br>Por favor selecione a Nave em que seu amigo esta.<br><br><yellow>■para aqueles que ja jogam:<c><br>Selecione a Nave em que ja joga."""
```

Comment:
The text in "FirstPlay_Message#0" is broken
\""Nave""<c>
you need to change this to:
\""Nave\""<c>

Reply:
Fixed

Reply:
Sorry, this is not fixed...

Reply:
yeh i noticed my mistake after i did it again, this line should me fine now

Reply:
all the changes needed are done 
ready for posting